clize/parser.py

```diff
@@ -232,15 +240,27 @@ def is_true(arg):
     six.binary_type: identity,
 }
 
+if pathlib:
+    @value_converter(name='PATH')
+    def path_converter(arg):
+        return pathlib.Path(arg)
+
+    _implicit_converters[pathlib.PurePath] = path_converter
+
 
 def get_value_converter(annotation):
     try:
         return _implicit_converters[annotation]
     except KeyError:
         pass
-    if not getattr(annotation, '_clize__value_converter', False):
-        raise ValueError('{0!r} is not a value converter'.format(annotation))
-    return annotation
+    if getattr(annotation, '_clize__value_converter', False):
+        return annotation
+    if not isinstance(annotation, type):
+        annotation = type(annotation)
```

Comment:
This makes it so that any callable (see line 827)'s type, instead of types themselves would be considered a converter. This seems unnecessarily broad. Is there a reason for it?

This is already the behavior for default parameters, by the way.

Reply:
I put it there because of a test not passing but it seems I misunderstood what to do.

The `issubclass` call expect a type as argument, and I thought an instance was being passed. Instead it was a callable that should raise a ValueError.

So instead of this, I only run the fallback if the annotation is a type.